include/types.h

```diff
@@ -166,11 +166,15 @@ typedef s32 DrawingLayer;
 typedef s16 PaintingData;
 typedef s32 CameraTransitionAngle;
 typedef s16 Movtex;
+typedef s16 MacroObject; // Required for backwards compatibility with Fast64
 typedef s16 Trajectory;
 typedef u8  CutsceneID;
 
 typedef u8 uchar;
 
+// Required for backwards compatibility with Fast64
+#define MACRO_OBJECT_END() 0
```

Comment:
This probably wants to exist in a different file

Reply:
putting it in the other file didnt work because it isnt included, this macro is exported to the same place as the MacroObject typedef is used so this is the most reliable place to put it

Reply:
It isn't including the location it used to exist in? That doesn't make sense.

Reply:
Wait did that PR also kill off collision objects?

Reply:
yeah arctic we tested putting it on the file that you're thinking of, but it didn't work. it has to be on this file for it to work. so if that was the only problem you see can you approve pls :(

Reply:
> It isn't including the location it used to exist in? That doesn't make sense.

this macro was originally in a tiny header file that only contained macro/special object defines so that file was completely deleted